draft-ietf-quic-qpack.md

```diff
@@ -188,30 +188,26 @@ input header list.
 QPACK is designed to contain the more complex state tracking to the encoder,
 while the decoder is relatively simple.
 
-### Reference Tracking
-
-An encoder MUST ensure that a header block which references a dynamic table
-entry is not processed by the decoder after the referenced entry has been
-evicted.  Hence the encoder needs to retain information about each compressed
-header block that references the dynamic table until that header block is
-acknowledged by the decoder; see {{header-acknowledgement}}.
-
 ### Limits on Dynamic Table Insertions {#blocked-insertion}
 
 Inserting entries into the dynamic table might not be possible if the table
 contains entries which cannot be evicted.
 
 A dynamic table entry cannot be evicted immediately after insertion, even if it
 has never been referenced. Once the insertion of a dynamic table entry has been
-acknowledged and there are no outstanding unacknowledged references to the
-entry, the entry becomes evictable.
+acknowledged and there are no outstanding references to the entry in
+unacknowledged header blocks, the entry becomes evictable.  Note that
+unacknowledged references on the encoder stream do not preclude the eviction of
```

Comment:
I don't agree with introducing the concept of "an unacknowledged reference on the encoder stream."  In the current version of the draft (12), the term "unacknowledged reference" always implies a reference from the header block.  In addition, only insertions and header blocks can be acknowledged.

Reply:
I agree with your observation, thank you.  I removed "unacknowledged" from this sentence.  Does it read better?

Reply:
This might be a (very old!) holdover from when inserts weren't always on a single stream.  In that case, you might have an unacknowledged insert that depended on another entry for the field name.  But as they *are* on a single stream, the insert that causes an eviction can only arrive after the insert that has the dependency has already been processed.